dom0/sd-dom0-qvm-rpc.sls

```diff
@@ -17,6 +17,8 @@ dom0-rpc-qubes.ClipboardPaste:
     - marker_start: "### BEGIN securedrop-workstation ###"
     - marker_end: "### END securedrop-workstation ###"
     - content: |
+        @tag:send-clipboard-to-sd @tag:sd-workstation ask
+        @tag:sd-workstation @tag:receive-clipboard-from-sd ask
```

Comment:
Pleased to report that the directionality appears to work as intended: it's possible to allow clipboard pastes _into_ e.g. `sd-app`, while still forbidding pastes out. Testing is critical here, will likely take the bulk of review time. Fleshing out the test plan with specific targets, even more verbose than already written, is likely to help coordinate testing.

Reply:
Test plan is plenty verbose now :)